src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -41,7 +41,7 @@ import { divePacketIconStyle, driftPacketIconStyle, getRallyStyle } from './shar
 import { createBotCourseOverGroundFeature, createBotHeadingFeature } from './shared/BotFeature'
```

Comment:
For consistency on the github page let start using the branch name as the pull request name "feature/taskpacket-dive-show-depth-value-only-if-bottom-strike"